instrumentation/netty/netty-4.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_1/NettyServerSingletons.java

```diff
@@ -3,24 +3,20 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.javaagent.instrumentation.netty.v4_1.server;
+package io.opentelemetry.javaagent.instrumentation.netty.v4_1;
```

Comment:
same here

Reply:
[Same reasoning](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/6820#discussion_r989477656). Don't feel strongly about this, BTW.